09-uploads-downloads.Rmd

```diff
@@ -4,10 +4,10 @@
 knitr::opts_chunk$set(eval = FALSE)
 ```
 
+## 9.4 Exercises {.unnumbered}
```

Comment:
Is {.unnumbered} the same as {-}?

Reply:
I dont remember I did this, it is likely got edited automatically when I switched to "visual markdown editor" mode in rstudio. Do you want me to change it back?

Reply:
Yea, I have changed it back.